frontend/src/components/Pages/MissionDefinitionPage/MissionDefinitionPage.tsx

```diff
@@ -48,21 +62,29 @@ const StyledMissionDefinitionPage = styled.div`
     gap: 1rem;
     margin: 2rem;
 `
+const StyledButton = styled(Button)`
+    width: 150px;
+`
+const StyledInspectionFrequencyDiv = styled.div`
+    > * {
+        padding: 10px;
+    }
+`
 
-interface IProps {
-    missionDefinition: CondensedMissionDefinition
-    updateMissionDefinition: (missionDefinition: CondensedMissionDefinition) => void
-}
-
-function KeyValuePairDisplay({ left, right }: { left: string; right: any }) {
+function KeyValuePairDisplay({ left, right, onEdit }: { left: string; right: any; onEdit?: () => void }) {
```

Comment:
Suggest renaming left and right as they aren't actually left and right. 
Maybe something like "key" and "value" or "heading" and "content"